.pre-commit-config.yaml

```diff
@@ -42,3 +42,8 @@ repos:
         args: ["--rcfile=sdks/python/.pylintrc"]
         files: ^sdks/python/apache_beam/
         exclude: *exclude
+  - repo: https://github.com/pycqa/isort
```

Comment:
Added as CI runs isort but pre-commit doesn't. Should have same behavior as ci. The setup.cfg tweaks I did were to match run_pylint.sh in CI.